testsuites/gpu/gpusuite.py

```diff
@@ -0,0 +1,89 @@
+from assertpy import assert_that
+
+from pathlib import Path
+
+from lisa import Environment, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.executable import CustomScript, CustomScriptBuilder
+from lisa.testsuite import simple_requirement
+from lisa.util import commonlib
+from lisa.tools import Lscpu, Lsvmbus
+
+@TestSuiteMetadata(
+    area="gpu",
+    category="functional",
+    description="""
+    This test suite runs the gpu test cases.
+    """,
+    tags=[],
+)
+class gpu(TestSuite):
+
+    unsupported_distro = ["CLEARLINUX", "COREOS"]
```

Comment:
There are not popular distros, why they are specified here as a denied list? Can we have an allowed list?

Reply:
Hmm So I thought there are only these few which are not supported, wanted to have the shorter list to validate against. If the supported distros are fewer then yeah make sense to have allow list.